packages/runtime/container-runtime/src/id-compressor/idCompressor.ts

```diff
@@ -80,15 +79,21 @@ export class IdCompressor implements IIdCompressor, IIdCompressorCore {
 	// The gen count to be annotated on the range returned by the next call to `takeNextCreationRange`.
 	// This is updated to be equal to `generatedIdCount` + 1 each time it is called.
 	private nextRangeBaseGenCount = 1;
-	// The capacity of the next cluster to be created
-	private newClusterCapacity = initialClusterCapacity;
 	private readonly sessions = new Sessions();
 	private readonly finalSpace = new FinalSpace();
 
 	// -----------------------
 
-	// ----- Telemetry state -----
+	// ----- Ephemeral state -----
 
+	/**
+	 * Roughly equates to a minimum of 1M sessions before we start allocating 64 bit IDs.
```

Comment:
Nit: this explains some details about how this property is consumed, etc., but doesn't describe its semantics. Would be worth documenting those, and probably moving the rest of these details into a `@remarks` block.

Reply:
I had the same thought as above, in particular about why 512 correlates with allocating 1 million 32 bit ids. 

I'll add that you can avoid making this public only for tests by (safely) accessing the property using bracket notation rather than dot notation ([example](https://www.typescriptlang.org/play?ssl=8&ssc=1&pln=9&pc=1#code/MYGwhgzhAEBiD29oG8CwAoAkABwE4EsA3MAFwFNoxoBeaAIjDoG4MBfDDAek+jN13i4MweADsIJaAA8a0UWQDucRAAoAlADowLdFx6ikfAUPQjxkgJ6z5ShPHUBtBnQC6OoA)). Doesn't seem to be too big a deal here, since this code isn't used by consumers AFAICT, but figured I'd mention it if you want to remove the comment.

Reply:
Wait - what? You can just do that? Access any private member of any class via index signature? It even auto-suggests the private property names and returns the correct type. I had no idea lol. I thought that in order to access TS-private class members you always needed to do unsafe casts, or use reflection or JS property APIs. 

This makes using #private, in general, much more of an attractive proposal.

Reply:
Yeah it's an explicit and intentional escape hatch blessed by typescript from my understanding. It's type-safe and has autocomplete. I use it all the time to write [tests that verify internals](https://github.com/microsoft/FluidFramework/blob/main/packages/dds/merge-tree/src/partialLengths.ts#L924).

Two caveats:

If you don't have `noImplicitAny` enabled, you won't get an error for invalid accesses like `new Foo()["b']` in the above example. In the playground they have this enabled so it's an error, but at least in the case of sequence we don't have this enabled. It actually looks like the only DDS with this enabled is `tree2` (I'll see if I can change this).

The other is that our default linter configuration warns against this. This is trivial to get around (ctrl+. => down arrow => enter), but does discourage its use in code outside of tests.

Reply:
This is great. I made it private!